src/main/scala/com/amazon/deequ/suggestions/rules/RetainCompletenessRule.scala

```diff
@@ -25,17 +25,20 @@ import scala.math.BigDecimal.RoundingMode
   * If a column is incomplete in the sample, we model its completeness as a binomial variable,
   * estimate a confidence interval and use this to define a lower bound for the completeness
   */
-case class RetainCompletenessRule() extends ConstraintRule[ColumnProfile] {
+case class RetainCompletenessRule(
+  minimumCompleteness: Double = 0.2,
+  maximumCompleteness: Double = 1.0,
+  z: Double = 1.96
```

Comment:
This PR looks great overall! Two minor asks: could you rename the z parameter to something more descriptive (e.g. `sensitivity` or similar). Also, could you add parameter descriptions to the class' docstring?